data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/mutate/CopyValueConversion.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.logstash.mapping.mutate;
+
+import java.util.List;
+
+public class CopyValueConversion extends AbstractConversion<CopyValueConversion.CopyValueConfig> {
+    public static class CopyValueConfig {
+        public final String from_key;
+        public final String to_key;
+
+        public CopyValueConfig(final String from_key, final String to_key) {
```

Comment:
Please use Java conventions: `fromKey`, `toKey`

Reply:
I have to have it in this form so that it serializes correctly in the yaml file